src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -386,25 +394,35 @@ sub-<label>/[ses-<label>/]
        sub-<label>[_ses-<label>][_acq-<label>][_dir-<label>][_run-<index>]_sbref.json
 ```
 
-Diffusion-weighted imaging data acquired for that participant. The OPTIONAL
-[`acq-<label>`](../99-appendices/09-entities.md#acq)
+The OPTIONAL [`acq-<label>`](../99-appendices/09-entities.md#acq)
 key/value pair corresponds to a custom label the user may use to
-distinguish different set of parameters. For example this should be used when a
-study includes two diffusion images - one single band and one multiband. In such
-case two files could have the following names:
+distinguish different sets of parameters.
+
+### Combining multi- and single-band acquisitions
+
+The single-band reference image MAY be stored as type `sbref` (e.g.,
+`dwi/sub-control01_sbref.nii[.gz]`,) as long as the image has no corresponding
+gradient information (`.bval` and `.bvec` sidecar files) to be stored.
+
+Otherwise, if some gradient information is associated to the single-band diffusion
+image and a multi-band diffusion image also exists,  the `acq-<label>` key/value pair
+MUST be used to distinguish both images.
+In such case two files could have the following names:
 `sub-01_acq-singleband_dwi.nii.gz` and `sub-01_acq-multiband_dwi.nii.gz`,
 however the user is free to choose any other label than `singleband` and
-`multiband` as long as they are consistent across subjects and sessions. For
-multiband acquisitions, one can also save the single-band reference image as
-type `sbref` (e.g. `dwi/sub-control01_sbref.nii[.gz]`) The bvec and bval files
-are in the [FSL format](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide#DTIFIT):
+`multiband` as long as they are consistent across subjects and sessions.
+
+### REQUIRED gradient orientation information
+
+The bvec and bval files are in the
+[FSL format](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide#DTIFIT):
 The bvec files contain 3 rows with n space-delimited floating-point numbers
-(corresponding to the n volumes in the relevant NIfTI file). The first row
-contains the x elements, the second row contains the y elements and third row
-contains the z elements of a unit vector in the direction of the applied
-diffusion gradient, where the i-th elements in each row correspond together to
-the i-th volume with `[0,0,0]` for non-diffusion-weighted volumes. Inherent to
-the FSL format for bvec specification is the fact that the coordinate system of
+(corresponding to the n volumes in the relevant NIfTI file).
+The first row contains the *x* elements, the second row contains the *y* elements and
+the third row contains the *z* elements of a unit vector in the direction of the applied
+diffusion gradient, where the *i*-th elements in each row correspond together to
+the *i*-th volume, with `[0,0,0]` for non-diffusion-weighted volumes.
+Inherent to the FSL format for bvec specification is the fact that the coordinate system of
```

Comment:
The FSL bvec system is a more complicated than this. It is only true [for some image orientations](https://www.mrtrix.org/2016/04/15/bug-in-fsl-bvecs-handling/). BIDS will need to decide whether they really want to be 100% FSL-compatible or stick to the image axis convention.

Reply:
Totally agree. My plan is to rebase your #352 on this after this is accepted (I believe this refactor should be less controversial), and then push that PR to the endline.